hazelcast/src/test/java/com/hazelcast/jet/impl/connector/HazelcastRemoteConnectorTest.java

```diff
@@ -70,7 +71,7 @@
 import static org.junit.Assert.assertTrue;
 
 @RunWith(HazelcastSerialClassRunner.class)
-@Category(SerialTest.class)
+@Category({QuickTest.class, ParallelJVMTest.class})
```

Comment:
This shouldn't be parallel jvm because of Hazelcast.newHazelcastInstance(

Reply:
Ok.